src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,17 @@
+import { useState } from "react";
+import "./checkbox.scss";
+
+export default function Checkbox() {
+  const [checked, setChecked] = useState(false);
+
+  function toggle() {
+    setChecked(!checked);
+  }
+
+  const checkedClass = checked ? "checkbox--checked" : "";
+  const containerClass = `checkbox ${checkedClass}`.trim();
+
+  return (
+    <input className={containerClass} onClick={toggle} type="checkbox"></input>

```

Comment:
You can self close your tag if doesn't have anything inside of it 

```suggestion
    <input className={containerClass} onClick={toggle} type="checkbox" />
```